aiida/engine/processes/calcjobs/manager.py

```diff
@@ -135,16 +156,16 @@ def _update_job_info(self):
 
     @contextlib.contextmanager
     def request_job_info_update(self, job_id):
-        """
-        Request job info about a job when it next changes it's job state.  If the job is not
-        found in the jobs list at the update the future will resolve to None.
+        """Request job info about a job when it next changes it's job state.
+
+        If the job is not found in the jobs list at the update the future will resolve to `None`.
 
-        :param job_id: The job identifier
-        :return: A future that will resolve to a JobInfo object when the job changes state
+        :param job_id: job identifier
+        :return: future that will resolve to a `JobInfo` object when the job changes state
         """
         # Get or create the future
         request = self._job_update_requests.setdefault(job_id, concurrent.Future())
-        assert not request.done(), "The future should be no be in the done state"
+        assert not request.done(), 'The future should be no be in the done state'
```

Comment:
could this be more explicit? e.g. something like
```suggestion
        assert not request.done(), 'Expected pending job info future, found in done state.'
```